Makefile

```diff
@@ -44,6 +48,7 @@ gen-golden-packages:
 gen-golden-all: gen-golden-component gen-golden-packages ## For Renovate
 
 .PHONY: clean-all
```

Comment:
Not quite related to the change here but:

I've just seen that I still have a left-over kind cluster from testing this. So I wanted to clean it up.

Can we change this to just `clean`? We've used the target `clean` in most other AppCat projects.

Reply:
Changed to `clean`